app/views/local_transactions/_fields.html.erb

```diff
@@ -48,6 +48,41 @@
           <%= f.text_area :need_to_know, rows: 4, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
         </span>
       </div>
+
+      <% administrations = {
+        "Scotland" => :scotland_availability,
+        "Wales" => :wales_availability,
+        "Northern Ireland" => :northern_ireland_availability
+      } %>
+      <% administrations.each do |title, field_name| %>
+        <%= f.fields_for field_name do |f| %>
+          <p class="h4"><%= title %></p>
+          <div class="form-group">
+            <div class="form-wrapper radio">
+              <%= f.label :type_local_authority_service, class: "control-label" do %>
+                <%= f.radio_button :type, 'local_authority_service', disabled: @resource.locked_for_edits? %>
+                Service available from local council
+              <% end %>
+            </div>
+            <div class="form-wrapper radio">
+              <%= f.label :type_devolved_administration_service, class: "control-label" do %>
+                <%= f.radio_button :type, 'devolved_administration_service', disabled: @resource.locked_for_edits? %>
+                Service available from devolved administration (or a similar service is available)
+              <% end %>
+            </div>
+            <div class="form-wrapper inset-text-field">
+              <%= f.label :alternative_url, "Enter the URL of the devolved administration website page", class: "non-bold-label" %>
+              <%= f.text_field :alternative_url, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
+            </div>
```

Comment:
```suggestion
            <div class="form-group inset-text-field">
              <span class="form-label">
                <%= f.label :alternative_url, "Enter the URL of the devolved administration website page", class: "non-bold-label" %>
              </span>
              <span class="form-wrapper">
                <%= f.text_field :alternative_url, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
              </span>
            </div>
```

Just a suggestion of how to make this field closer in element structure to existing fields. I'm anticipating we'll want this consistency when it comes to fixing error handling.

Reply:
Totally agree - a consistent approach would be the best approach for error handling. Fixed up as per your suggestion. I've also added the missing `<div class="form-group">`'s around each form element - for consistency.